package-system/pyside2/Findpyside2.cmake.Linux

```diff
@@ -25,11 +25,38 @@ set(${MY_NAME}_RUNTIME_DEPENDENCIES
     ${PYSIDE_BASE_PATH}/lib/libshiboken2.abi3.so.5.15
     ${PYSIDE_BASE_PATH}/lib/libshiboken2.abi3.so
 )
+
 ly_add_target_files(TARGETS ${TARGET_WITH_NAMESPACE} FILES ${${MY_NAME}_RUNTIME_DEPENDENCIES})
 
 # In addition to the libraries, create a Pyside2::Tools library that handles the shiboken and pyside2-lupdate executables that are part o
 # of the pyside2 package
```

Comment:
opportunity to fix the typo here (the floating o)

Reply:
Yes, combining them works. For testing I've been overwriting the Findpyside.cmake file in my existing 3p pyside package.

Reply:
Merged.